judoscale-ruby/lib/judoscale/request_metrics.rb

```diff
@@ -20,15 +24,23 @@ def started_at
       if @request_start_header
         # There are several variants of this header. We handle these:
         #   - whole milliseconds (Heroku)
+        #   - whole microseconds (???)
```

Comment:
Lol what?

Reply:
Prior to this PR, my code comment said "nanoseconds", but my implementation handled "microseconds". I'm not even sure which one Render uses. I can't find it documented anywhere.

Reply:
Well dang.

[<img width="674" alt="image" src="https://user-images.githubusercontent.com/8585984/225033131-186e25b9-09dc-44b9-873a-011a8a29deaa.png">](https://stackoverflow.com/a/24106694/7552158)

But I think we're well within safe constraints. Interesting they don't mention it anywhere though. Maybe they don't anticipate people using that header